identity/identity_verification.go

```diff
@@ -57,7 +57,7 @@ type VerifiableAddress struct {
 	// enum: ["email"]
```

Comment:
```suggestion
	// enum: ["email", "sms"]
```
or remove this.
The generated SDK probably still respects this comment.